local-dev/indexing-scripts/index-mycocosm.rb

```diff
@@ -0,0 +1,42 @@
+require './services'
```

Comment:
Why are we using ruby for these test scripts.  Why introduce yet another language in the middle of this?